tools/rsdl/rsdl-js/package-lock.json

```diff
@@ -1,3861 +0,0 @@
-{
```

Comment:
Do you really want to delete this? It belongs to a different package.

Reply:
Didn't mean to remove permanently; I was getting a bunch of errors trying to build that went away when I deleted this file.  I can try and restore and fix the issues, but removing the file was the most expedient way to get back to the issue at hand.

Reply:
In such situations just delete it and run `npm install` again to restore a better version of it.